src/quiz/question.jsx

```diff
@@ -0,0 +1,67 @@
+/*This QuizH component shows the whole section number 2 of the questions */
+const QuizH = ({ arr }) => {
+  const il = arr.option.map((val) => {
+    return (
+      <>
+        <li>
+          <label for={val.id}>
+            <input
+              type="radio"
+              id={val.id}
+              name={val.nameInput}
+              value={val.value}
+            />
+            {val.label}
+          </label>
+        </li>
+      </>
+    );
+  });
+  return (
+    <div class="question-block">
+      <p>{arr.num}</p>
+      <fieldset className="question" name={arr.nameField}>
+        <legend>{arr.legend}</legend>
+        <ul class="answers-list">{il}</ul>
+      </fieldset>
+    </div>
+  );
+};
+
+/* This HTML component selection number 2 of the page  */
+export const Question = () => {
+  /* Object array for each div, fieldset and legend*/
+  const object = [
```

Comment:
this is not an object but an array, and it contains the data about questions, for clarity we could name it `questionsArray`, since it's also static data we could move outside of the component